packages/eslint-plugin-orbit-components/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "eslint-plugin-orbit-components",
+  "version": "0.2.0",
+  "description": "Collection of ESLint rules to enforce the best usage practices of @kiwicom/orbit-components",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kiwicom/eslint-plugin-orbit-components"
+  },
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin",
+    "orbit-components",
+    "kiwicom"
+  ],
+  "author": "Kiwi.com",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "rm -fr ./dist && babel ./src --extensions '.ts' --out-dir ./dist",
```

Comment:
Setting aside the question whether to have the build step at all, if we're going to have it we also need a `prepublishOnly` script which runs `build` because the root `publish-packages` script runs `publish` in every package, so if some packages need to run anything before publishing, they need to define it for themselves.